src/lib/operators/abstract_join_operator.cpp

```diff
@@ -10,15 +10,15 @@ namespace opossum {
 
 AbstractJoinOperator::AbstractJoinOperator(const std::shared_ptr<const AbstractOperator> left,
                                            const std::shared_ptr<const AbstractOperator> right, const JoinMode mode,
-                                           const std::pair<ColumnID, ColumnID>& column_ids, const ScanType scan_type)
+                                           const JoinColumnIDs& column_ids, const ScanType scan_type)
```

Comment:
I believe we wanted to call this ColumnIdPair

Reply:
... `ColumnIDPair` since the D in ColumnID is capital?...

Reply:
sure